.drone.yml

```diff
@@ -17,6 +17,19 @@ pipeline:
       matrix:
         NEED_CORE: true
 
+  install-testrunner:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - git clone -b stable10 --depth=1 https://github.com/owncloud/core.git /var/www/owncloud/testrunner
+      - git clone https://github.com/owncloud/activity.git /var/www/owncloud/testrunner/apps/activity
```

Comment:
Why do we need the activity app cloned?

Reply:
We are running test for ```activity``` app from the same drone of ```files_texteditor``` app. So, we need ```activity``` app in our testrunner.